hazelcast-coverage-report/pom.xml

```diff
@@ -0,0 +1,294 @@
+<!--
+  ~ Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>hazelcast-root</artifactId>
+        <groupId>com.hazelcast</groupId>
+        <version>5.3.0-SNAPSHOT</version>
```

Comment:
Incorrect version.

Reply:
Thanks, fixed in 9d4604d8bf39776370be343b7ad662a8f3fdca33 (set to `5.2.2-SNAPSHOT`)